example/imagery/src/ImageTelemetry.js

```diff
@@ -31,10 +31,11 @@ define(
 
         var firstObservedTime = Date.now(),
             images = [
-                "http://www.nasa.gov/393811main_Palomar_ao_bouchez_10s_after_impact_4x3_946-710.png",
-                "http://www.nasa.gov/393821main_Palomar_ao_bouchez_15s_after_impact_4x3_946-710.png",
-                "http://www.nasa.gov/images/content/393801main_CfhtVeillet2_4x3_516-387.jpg",
-                "http://www.nasa.gov/images/content/392790main_1024_768_GeminiNorth_NightBeforeImpact_946-710.jpg"
+                "http://www.esa.int/var/esa/storage/images/esa_multimedia/images/2015/01/comet_on_16_january_2015_navcam/15205508-1-eng-GB/Comet_on_16_January_2015_NavCam.jpg",
+                "http://www.esa.int/var/esa/storage/images/esa_multimedia/images/2014/08/comet_on_7_august_a/14721226-1-eng-GB/Comet_on_7_August_a_node_full_image_2.jpg",
+                "http://http://resources0.news.com.au/images/2014/10/26/1227102/619948-e62d0d0c-5cb3-11e4-9c68-d403f5dcc36d.jpg",
+                "http://www.esa.int/var/esa/storage/images/esa_multimedia/images/2014/08/comet_on_16_august_a/14735866-1-eng-GB/Comet_on_16_August_a_node_full_image_2.jpg",
+			  	"http://www.esa.int/var/esa/storage/images/esa_multimedia/images/2015/08/year_at_a_comet_may_2015/15549436-1-eng-GB/Year_at_a_comet_May_2015_node_full_image_2.jpg"
             ].map(function (url, index) {
```

Comment:
My bad: inadvertent. I've restored the old URLs and pushed the change.
________________________________
From: Victor Woeltjen [notifications@github.com]
Sent: Tuesday, September 22, 2015 4:31 PM
To: nasa/openmctweb
Cc: Hacskaylo, Charles F. (ARC-TI)[SGT, INC]
Subject: Re: [openmctweb] Review and integrate open95b into master (#118)


In example/imagery/src/ImageTelemetry.js<https://github.com/nasa/openmctweb/pull/118#discussion_r40157688>:

> @@ -31,10 +31,11 @@ define(
>
>          var firstObservedTime = Date.now(),
>              images = [
> -                "http://www.nasa.gov/393811main_Palomar_ao_bouchez_10s_after_impact_4x3_946-710.png",
> -                "http://www.nasa.gov/393821main_Palomar_ao_bouchez_15s_after_impact_4x3_946-710.png",
> -                "http://www.nasa.gov/images/content/393801main_CfhtVeillet2_4x3_516-387.jpg",
> -                "http://www.nasa.gov/images/content/392790main_1024_768_GeminiNorth_NightBeforeImpact_946-710.jpg"
> +                "http://www.esa.int/var/esa/storage/images/esa_multimedia/images/2015/01/comet_on_16_january_2015_navcam/15205508-1-eng-GB/Comet_on_16_January_2015_NavCam.jpg",
> +                "http://www.esa.int/var/esa/storage/images/esa_multimedia/images/2014/08/comet_on_7_august_a/14721226-1-eng-GB/Comet_on_7_August_a_node_full_image_2.jpg",
> +                "http://http://resources0.news.com.au/images/2014/10/26/1227102/619948-e62d0d0c-5cb3-11e4-9c68-d403f5dcc36d.jpg",
> +                "http://www.esa.int/var/esa/storage/images/esa_multimedia/images/2014/08/comet_on_16_august_a/14735866-1-eng-GB/Comet_on_16_August_a_node_full_image_2.jpg",
> +                             "http://www.esa.int/var/esa/storage/images/esa_multimedia/images/2015/08/year_at_a_comet_may_2015/15549436-1-eng-GB/Year_at_a_comet_May_2015_node_full_image_2.jpg"
>              ].map(function (url, index) {


@charlesh88<https://github.com/charlesh88> Were these changes piggybacked intentionally or uninentionally? If we do want to keep these I'd like to remove the one that's to a non-NASA/non-ESA maintained site (strictly due to a general paranoia about checking in URLs, particularly to code that may run in the client)

—
Reply to this email directly or view it on GitHub<https://github.com/nasa/openmctweb/pull/118/files#r40157688>.